certified-connectors/DocuSignDemo/script.csx

```diff
@@ -712,6 +712,14 @@ public class Script : ScriptBase
     var uriLogicAppsBase64 = Convert.ToBase64String(Encoding.UTF8.GetBytes(uriLogicApps ?? string.Empty));
     var notificationProxyUri = this.Context.CreateNotificationUri($"/webhook_response?logicAppsUri={uriLogicAppsBase64}");
 
```

Comment:
add a comment here indicating that this is for backward compatibility. to be removed once we deprecate this.

Reply:
done